### PR TITLE
PRs ready for review.

### PR DESCRIPTION
**Pull request by Jasoneto**
I added new styles to the homepage like margin and padding in the tags p and h2, with the respetive BEM classes.

### CODE REVIEW

**Review by Theweeksad — APPROVED** [review]
nice!!

File: CSS/normalice.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+main {
+    display: block;
+}
+
+h1 {
+    font-size: 2em;
+    margin: 0.67em 0;
+}
```
Comment (johanmendezb) [inline CSS/normalice.css]: Please remove this `h1` properties.

**Review by johanmendezb — CHANGES_REQUESTED** [review]
please review my comment, gj, thanks!

**Review by Theweeksad — APPROVED** [review]
nice!

**Review by johanmendezb — APPROVED** [review]

**Pull request merged**